tests/test_diffeq/test_odefiltsmooth/test_odefiltsmooth.py

```diff
@@ -131,7 +131,7 @@ def setUp(self):
         self.ivp = ode.logistic([0.0, 1.5], initrv)
         self.step = 0.5
         sol = probsolve_ivp(self.ivp, step=self.step, diffconst=1.0, which_prior="ibm1")
-        state_rvs = sol._state_rvs
+        state_rvs = sol.kalman_posterior.state_rvs
```

Comment:
the kalman_posterior is a public attribute now. (1) one is free to change it; it does not break the implementation: (2) it avoids the awkward state_rv wrapping.